app/assets/stylesheets/fonts.scss

```diff
@@ -1,19 +1,43 @@
 $brand: "Comfortaa", "Helvetica", sans-serif;
+$body-text: "Fira Sans", "Helvetica", sans-serif;
 
 .monospace{
   font-family: "Fira Mono", monospace;
   font-size: 1.1rem;
 }
 
+.brand {
+  font-family: $brand;
+}
+
+
+
 html {
-  font-family: "Fira Sans", "Helvetica", sans-serif;
+  font-family: $body-text;
 }
 
-.brand {
-  font-family: $brand;
+h1 {
+  @extend .brand;
+  text-align: center;
+}
+h2 {
+  @extend .dark-text;
+}
+h3 {
+  @extend .dark-text;
+}
+h4{
+  @extend .dark-text;
+}
+h5{
+
+}
+h6{
+  
 }
 
 p {
+  @extend .dark-text.secondary;
```

Comment:
Remember the MD specs are calibrated for Google's in house font Roboto and particularly to be read on mobile devices. So what works for them might need to be tweaked for us. Overall the changes look good though so I'll merge into dev for now.